ansible/server-state-playbooks/nightshade-web/playbook.yml

```diff
@@ -0,0 +1,173 @@
+# Install OMERO.web with a public user on localhost
+
+- hosts: all
+
+  pre_tasks: 
+
+    # For OMERO.web apps
+    # Pythonpath must exist before omero.web.config is provisioned
+    # or OMERO.web won't start.
+    # create systemd file addition for PYTHONPATH
+    - name: OMERO.web apps | (pre_task) configure systemd for pythonpath - create config folder 
+      become: yes
+      file:
+        dest: "/etc/systemd/system/omero-web.service.d"
+        state: directory
+        mode: "u=rwx,go=rx" 
+        owner: "root" 
+        group: "root" 
+
+    # systemd web-apps folder to pythonpath to env
+    - name: OMERO.web apps | (pre_task) configure systemd for pythonpath 
+      become: yes
+      blockinfile:
+        create: yes
+        destfile: /etc/systemd/system/omero-web.service.d/pythonpath.conf
+        owner: "root" 
+        group: "root" 
+        block: |2+
+          [Service]
+          Environment="PYTHONPATH=$PYTHONPATH:/opt/omero/web/web-extensions"
+      notify:
+        - reload systemd
+
+  roles:
+
+    # Root LV Size 
+    - role: openmicroscopy.lvm-partition
+      lvm_lvname: "{{ provision_root_lvname }}"
+      lvm_vgname: "{{ provision_root_vgname }}"
+      lvm_lvmount: /
+      lvm_lvsize: "{{ provision_rootsize }}"
+      lvm_lvfilesystem: "{{ provision_root_filesystem }}"
+
+    # OMERO.web configuration in host_vars in different repository
+    - role: openmicroscopy.omero-web
+      omero_web_release: 5.2.8
+
+    - role: openmicroscopy.system-monitor-agent
+      when: >
+            ((ansible_env.COBBLER_SERVER is defined)
+            and (ansible_env.COBBLER_SERVER == "spacewalk.lifesci.dundee.ac.uk"))
+
+  post_tasks:
+    - name: Install open-vm-tools if system is a VMware vm
+      become: yes
+      yum:
+        name: open-vm-tools
+        state: latest
+      when: >
+           ((ansible_virtualization_type is defined)
+           and (ansible_virtualization_type == "VMware"))
+
+    - name: NGINX - SSL File Deployment - prepare directory
+      become: yes
+      file:
+        path: "{{ nginx_ssl_files_path }}"
+        state: directory
+        owner: root
+        group: root
+        mode: "u=r,go=" 
+
+    - name: NGINX - SSL File Deployment
+      become: yes
+      copy:
+        dest="{{ item.key }}"
+        content="{{ item.value.content }}"
+        owner="{{ item.value.owner }}"
+        group="{{ item.value.group }}"
+        mode="{{ item.value.mode }}"
+      with_dict: "{{ nginx_ssl_cert_files }}"
```

Comment:
@manics - see https://dantehranian.wordpress.com/2015/07/24/managing-secrets-with-ansible-vault-the-missing-guide-part-1-of-2/ - it's exactly like this example.